hudi-common/src/main/java/org/apache/hudi/common/util/CompactionUtils.java

```diff
@@ -285,8 +285,7 @@ public static List<HoodieInstant> getPendingCompactionInstantTimes(HoodieTableMe
    */
   public static Option<Pair<HoodieTimeline, HoodieInstant>> getDeltaCommitsSinceLatestCompaction(
       HoodieActiveTimeline activeTimeline) {
-    Option<HoodieInstant> lastCompaction = activeTimeline.getCommitTimeline()
-        .filterCompletedInstants().lastInstant();
+    Option<HoodieInstant> lastCompaction = activeTimeline.filterCompletedCommitInstants().lastInstant();
```

Comment:
@nsivabalan : Can you rename getCommitTimeline() to getCommitAndReplaceTimeline() and then add an API getCommitOnlyTimeline(). 
The current naming of these APIs are very problematic and can easily cause bugs.

Reply:
And we better not reuse the action name for replace commit and compaction commit.